lib/index.d.ts

```diff
@@ -26,7 +27,7 @@ export interface IGlobPluginOptions {
      *   [acorn](https://www.npmjs.com/package/acorn) to check default exports.
      *   Intermediate files export both named exports and default exports.
      */
-    format?: IntermediateFileFormat,
+    format?: valueof<IntermediateFileFormat>,
```

Comment:
Note: This fixes the problem ↓

![note](https://user-images.githubusercontent.com/7359068/68633021-aea8f200-0533-11ea-835f-3b334b6768f4.png)

I will rewrite this to `export type = 'import' | 'named' ...` later.

Reply:
yup ` type = 'import' | 'named' ...` is probably more readable, but basically the same as `valueof<...>` .
Another solution would be to export the `IntermediateFileFormat` from your `js` file. Then, only those values could be used in the plugin when `format?: IntermediateFileFormat,` - a bit cumbersome but not entirely bad. Up to you :smile: 

 Would you mind releasing this little change soon ?

Reply:
I've released v0.4.5 that includes the fixes.